test/integration/tasks.test.js

```diff
@@ -30,6 +31,8 @@ describe("Tasks", function () {
     const superUserId = await addUser(superUser);
     jwt = authService.generateAuthToken({ userId });
     superUserJwt = authService.generateAuthToken({ userId: superUserId });
+    // Adding status to active user as while adding the user it will be consider as new user (for new user default status is idle)
```

Comment:
Is this required? Feels redundant to me

Reply:
After adding the user which already has status, why I am adding the status again. May feel confused at first. That's why I added comment